### DISCUSSION TITLE
我在启动服务时遇到了问题，请各位帮助

### DISCUSSION THREAD

**Discussion by MinecraftSTL** [RikkaApps/Shizuku#318] [Q&A]
我通过有线adb连接，然后看到了这些
![image](https://github.com/RikkaApps/Shizuku/assets/53145859/d3bfb53b-e5ff-4bf9-a69e-ce15c7845930)
我的电脑是win10 x64，手机是红魔8pro android13

**Comment by MinecraftSTL** [comment]
我再次仔细翻阅了帮助，帮助中的内容是否也代表只有android10以下的设备才能使用，而非只是Android10以下的设备必须使用

**Reply by xiaojinghan2022** [reply]
？你是不是多复制了一次
